NoteFind/Models/Book.h

```diff
@@ -15,10 +17,12 @@ NS_ASSUME_NONNULL_BEGIN
 @property (strong, nonatomic) NSURL *bookCoverURL;
 @property (strong, nonatomic) NSString *synposis;
 @property (strong, nonatomic) NSArray *authors;
+@property (strong, nonatomic) NSString *identifier;
```

Comment:
Traditionally, `NSString` types should use `copy` instead of `strong`.

> By default, object properties are strong, and scalar properties are assign. There's one exception: for properties that have a mutable counterpart, you want to define them as copy. For example, the name property is of type NSString. What could happen is that somebody creates a Person object and assigns a value of type NSMutableString. Then, sometime later, he or she might change the mutable string. If our property would have been strong instead of copy, our Person object would have changed too, which is not what we want. It's the same for containers, such as arrays or dictionaries.

https://www.objc.io/issues/7-foundation/value-objects/

Reply:
resolved, changes updated!